ui/app/reports/services/reportService.js

```diff
@@ -12,6 +13,14 @@ angular.module('bahmni.reports')
             "CUSTOM EXCEL": "application/vnd.ms-excel-custom",
             "ODS": "application/vnd.oasis.opendocument.spreadsheet"
         };
+        var avaialbleDateRange = {
+            "Today": currentDate,
+            "This Month": new Date(currentDate.getFullYear(), currentDate.getMonth(), 1),
```

Comment:
It appears that these options, like: today, this month, this quarter, etc don't come from translation file, but are hardcoded in code. Is this the case?

Reply:
Yes, its hardcoded like formats.

Reply:
Is it easy to move both into translation files? Ideally all UI visible strings should come from translation files?

Reply:
Not sure, how service will fetch data from translation file because its object  and how it will take key from translation file. I just tried that giving error `invalid key property`.

Reply:
ok. in that case, we can merge this PR as-is. And can raise a separate JIRA issue for enabling translation for these. Thanks!